tapir/accounts/templates/accounts/user_detail.html

```diff
@@ -22,38 +22,30 @@
     <div class="row">
         <div class="col-xl-6">
             <div class="card mb-2" id="tapir_user_detail_card">
-                <h5 class="card-header d-flex justify-content-between align-items-center">
+                <h5 class="card-header d-flex justify-content-between align-items-center flex-wrap gap-2">
                     <span>{% translate "Personal Data" %}</span>
-                    <span class="d-flex">
+                    <span class="d-flex justify-content-end flex-fill flex-wrap gap-2">
                         {% if perms.coop.admin %}
-                            <div class="ms-1">
```

Comment:
It looked like this wrapper div is not necessary but I can add it back if I missed something :)